src/core/loader/ncch.h

```diff
@@ -242,14 +236,22 @@ class AppLoader_NCCH final : public AppLoader {
     u32 core_version = 0;
     u8 priority = 0;
     u8 resource_limit_category = 0;
-    u32 ncch_offset = 0; // Offset to NCCH header, can be 0 or after NCSD header
+
     u32 exefs_offset = 0;
 
-    NCCH_Header ncch_header;
     ExeFs_Header exefs_header;
     ExHeader_Header exheader_header;
 
+protected:
     std::string filepath;
+    // Offset to NCCH header, can be 0 or after NCSD header
+    u32 ncch_offset = 0;
```

Comment:
i try to avoid same line comments because i sometimes evaluate them as referring to the line below that line